core/src/main/java/com/whylogs/core/ColumnProfile.java

```diff
@@ -31,28 +33,52 @@
 public class ColumnProfile {
   public static final int FREQUENT_MAX_LG_K = 7;
   private static final int CARDINALITY_LG_K = 12;
+  private static Map<String, String> ENV;
+  private static ImmutableList<String> nullEnvStrs;
 
   @NonNull private final String columnName;
   @NonNull private final CountersTracker counters;
   @NonNull private final SchemaTracker schemaTracker;
   @NonNull private final NumberTracker numberTracker;
   @NonNull private final ItemsSketch<String> frequentItems;
   @NonNull private final HllSketch cardinalityTracker;
+  private final ImmutableList<String> nullStrs;
+
+  static synchronized ImmutableList<String> nullStrsFromEnv() {
+    if (ColumnProfile.ENV == null) {
+      ColumnProfile.ENV = System.getenv();
+      String nullSpec = ColumnProfile.ENV.get("NULL_STRINGS");
+      ColumnProfile.nullEnvStrs =
+          nullSpec == null ? null : ImmutableList.copyOf(nullSpec.split(","));
+    }
+    return ColumnProfile.nullEnvStrs;
+  }
 
   public ColumnProfile(String columnName) {
+    this(columnName, nullStrsFromEnv());
+  }
+
+  public ColumnProfile(String columnName, ImmutableList<String> nullStrs) {
     this.columnName = columnName;
     this.counters = new CountersTracker();
     this.schemaTracker = new SchemaTracker();
     this.numberTracker = new NumberTracker();
     this.frequentItems = FrequentStringsSketch.create();
     this.cardinalityTracker = new HllSketch(CARDINALITY_LG_K);
+    this.nullStrs = nullStrs;
+  }
+
+  private boolean checkNullStrs(Object v) {
+    return this.nullStrs != null
+        && ((v instanceof String && this.nullStrs.contains(v))
+            || this.nullStrs.contains(v.toString()));
```

Comment:
you can just do `this.nullStrs.contains(v.toString())`. A `String` will return itself FYI